### DISCUSSION TITLE
How to get the Pull Shark badge?

### DISCUSSION THREAD

**Discussion by thesmilingsabre** [community/community#18610] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get the new Pull Shark GitHub achievement?

**Comment by brunomdrrosa** [comment, marked as answer]
To get the achievement, you need to open 2 pull requests that have been merged.
From the 16th PR, you win the bronze medal, after the 128th you win the silver medal and finally, in your 1024th you win the gold medal.

https://github.com/Schweinepriester/github-profile-achievements

**Reply by rapr-odoo** [reply]
So, is it like you will have to re-open PRs which has been already merged?

**Comment by danieldev23** [comment]
To get the achievement, you need to open 2 pull requests that have been merged.
From the 16th PR, you win the bronze medal, after the 128th you win the silver medal and finally, in your 1024th you win the gold medal.

https://github.com/danieldev23/github-profile-achievements
